### PR TITLE
Update rx_windows_user.h

### PR DESCRIPTION
**Pull request by freezestudio**
change WM_USRE+1 to WM

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@freezestudio__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, MSOTBOT;

File: Rx/v2/examples/win_text/rx_windows_user.h [hunk 1]
```diff
@@ -98,7 +98,7 @@ namespace rxcpp { namespace windows_user {
     {
         static LRESULT send_call(HWND w, std::function<LRESULT(Derived&)> f) {
             auto fp = reinterpret_cast<LPARAM>(std::addressof(f));
-            return SendMessage(w, WM_USER+1, 0, fp);
+            return SendMessage(w, WM, 0, fp);
```
Comment (kirkshoop) [inline Rx/v2/examples/win_text/rx_windows_user.h]: this is intended to send the WM template parameter. What issue did you encounter?

**Comment by kirkshoop** [comment]
Thank you for the PR!

File: Rx/v2/examples/win_text/rx_windows_user.h (hunk 1 above)
Comment (kirkshoop) [inline Rx/v2/examples/win_text/rx_windows_user.h]: oh, I inverted the change in my head. Nevermind.

**Pull request merged**